### ISSUE TITLE
Bug: Query: stringification not supported for node type collection

### ISSUE THREAD

**Issue by baslr** [arangodb/arangodb#2234]
ArangoDB 3.1.6
```
LET wuigerhojfoeru = FULLTEXT(authors, "name", "prefix:li")[*]._key

FOR doc IN authors

FILTER doc._key IN wuigerhojfoeru
SORT doc.name
RETURN doc.name
```
leads to `Query: stringification not supported for node type collection (exception location: arangod/Aql/AstNode.cpp:2186). Please report this error to arangodb.com (while optimizing plan) (exception location: arangod/RestHandler/RestCursorHandler.cpp:131). Please report this error to arangodb.com`

**Comment by jsteemann** [comment]
Fixed in 3.1.8 and devel.

**Comment by dothebart** [comment]
3.1.8 is available for download now.
